website/content/en/docs/deployment/cognito/manifest/guide-automated.md

```diff
@@ -164,8 +95,108 @@ make delete-kubeflow INSTALLATION_OPTION=helm DEPLOYMENT_OPTION=cognito
     ```
-    - Run the following command to install the script dependencies and delete the resources:
-    > Note: You can rerun the script incase some resources fail to delete
+
+## 2.0 Install Kubeflow
+
+Install Kubeflow using the following command:
+{{< tabpane persistLang=false >}}
+{{< tab header="Kustomize" lang="toml" >}}
+make deploy-kubeflow INSTALLATION_OPTION=kustomize DEPLOYMENT_OPTION=cognito
+{{< /tab >}}
+{{< tab header="Helm" lang="yaml" >}}
+make deploy-kubeflow INSTALLATION_OPTION=helm DEPLOYMENT_OPTION=cognito
+{{< /tab >}}
+{{< /tabpane >}}
+
+## 3.0 Update the domain with ALB address
+1. Check if ALB is provisioned. It takes about 3-5 minutes.
+    ```shell
+    kubectl get ingress -n istio-system
+    Warning: extensions/v1beta1 Ingress is deprecated in v1.14+, unavailable in v1.22+; use networking.k8s.io/v1 Ingress
+    NAME            CLASS    HOSTS   ADDRESS                                                              PORTS   AGE
+    istio-ingress   <none>   *       k8s-istiosys-istioing-xxxxxx-110050202.us-west-2.elb.amazonaws.com   80      15d
+    ```
+    If `ADDRESS` is empty after a few minutes, check the logs of  the `alb-ingress-controller` by following this [troubleshooting guide]({{< ref "/docs/troubleshooting-aws.md#alb-fails-to-provision" >}}).
+
+1. Substitute the ALB address under `kubeflow.alb.dns` in `tests/e2e/utils/cognito_bootstrap/config.yaml`. The kubeflow section of the config file will look like:
+    ```yaml
+    kubeflow:
+        alb:
+            dns: ebde55ee-istiosystem-istio-2af2-1100502020.us-west-2.elb.amazonaws.com
+            serviceAccount:
+                name: alb-ingress-controller
+                policyArn: arn:aws:iam::123456789012:policy/alb_ingress_controller_kube-eks-clusterxxx
+    ```
+
+1. Run the following script to update the subdomain with the ALB address:
+    ```shell
+    cd tests/e2e
+    PYTHONPATH=.. python utils/cognito_bootstrap/cognito_post_deployment.py
+    cd -
+        ```
+## 4.0 Connect to Kubeflow central dashboard 
+Follow the instructions in the `Connecting to Kubeflow central dashboard` section of the [Manual Deployment Guide]({{< ref "/docs/deployment/cognito/manifest/guide.md#60-connecting-to-central-dashboard" >}}) to:
+
+* Create a user in Cognito user pool.
+* Create a profile for the user from the user pool.
+* Connect to the central dashboard.
+
+## Uninstall Kubeflow
+
+> Note: Delete all the resources you might have created in your profile namespaces before running these steps.
+
+1. Run the following commands to delete the profiles, ingress and corresponding ingress managed load balancer
+   ```bash
+    kubectl delete profiles --all
+    ```
+
+1. Delete the kubeflow deployment
+
+    > Note: Make sure you have the correct INSTALLATION_OPTION and DEPLOYMENT_OPTION environment variables set for your chosen installation.
+    {{< tabpane persistLang=false >}}
```

Comment:
note can be removed since variables are explicitly being specified in the command